charts/vald/templates/agent/statefulset.yaml

```diff
@@ -107,5 +107,19 @@ spec:
       priorityClassName: {{ .Values.agent.name }}-priority
       {{- end }}
       {{- end }}
+{{- if .Values.agent.volumeStore }}
```

Comment:
maybe L110 and L123 are useless. (because default values.yaml always must have .Values.agent.volumeStore object)